### PR TITLE
Adds information on water supply and water markets

### PR DESCRIPTION
**Pull request by kvcalvin**
These are changes made by @skim301.

### CODE REVIEW

**Review by abigailsnyder — APPROVED** [review]

**Review by mwisepnnl — APPROVED** [review]

File: water.md [hunk 1]
```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
...
+Cooling system options compete in a calibrated logit nest, similar to the competition between fuels within this sector. The capital costs are from [NETL 2008](water.html#netl2008) and vary across cooling options. Electric generation efficiencies of dry-cooled power plants are lower than the other cooling options that utilize water. The competition between cooling system options is endogenous and cost-based. When water supply constraints are not imposed or when water prices and scarcity is low, the model output tends to largely reflect the exogenous share-weight assumptions, which follow the assumptions and logic of [Davies et al. 2013](water.html#davies2013). On othe hand, when water supply constraints are imposed and regional water scarcity escalates, the cost of water use directly affects the choice of cooling system.
```
Comment (pralitp) [inline water.md]: "On othe hand" should be "On the other hand"

File: water.md [hunk 2]
```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
...
+
+Renewable water supplies are based on runoff estimates from Xanthos, a detailed global hydrology model ([Liu et al. 2017](liu2017); [Vernon et al. 2019](vernon2019)). Xanthos accounts for surface and subsurface processes to compute runoff at 0.5° grid resolution. Global climate datasets are utilized in conjunction with Xanthos to determine historical annual average runoff aggregated for each basin ([Turner et al. 2019b](turner2019b); [Liu et al. 2017](liu2017); [Vernon et al. 2019](vernon2019)). Of the total basin runoff, water available or accessible for human use takes into consideration requirements for ecosystem services, inaccessibility due to rapid flow and remote locations, and capacity of reservoir storage. Accessible fractions of total runoff vary across basins. Renewable water volumes up to the accessible fraction is available at nominal or low cost. Additional renewable water beyond the accessible fraction is available at a sharply higher cost to ensure availability of water for ecosystem services and to reflect capital investments necessary for reservoir expansion.
```
Comment (pralitp) [inline water.md]: link syntax is not correct.  Should be `water.html#liu2017`, for instance

File: water.md [hunk 3]
```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
...
+The ability to combine and overlay multiple supply curves into one provides great flexibility in the potential for alternative characterization of water resources and their contributions to the overall supply. For instance, both renewable and non-renewable resources can be further disaggregated, such as by separating renewable water into surface and ground water (recharge) portions or separating fossil groundwater by alternative aquifer characteristics. The shape of the multiple resource curves and the degree to which they overlap or not can determine water use behavior, such as the simultaneous or sequential production of alternative water resources.
+
+One notable feature of the GCAM's Renewable Sub-Resource object is the ability to vary the maximum available resource potential by time period. This feature enables the assessment of climate impacts on water availability. Projected future changes in precipitation from climate models (GCMs) are utilized to calculate river runoff and changes in the maximum available potential of renewable water supplies by time and basin.
```
Comment (pralitp) [inline water.md]: This section and the next are generally more detailed description than we typically include in this form of documentation.  Which was by design.  That being said it leaves no really publicly available space for this level of documentation.  So I could be persuaded to leave it.

**Review by pralitp — CHANGES_REQUESTED** [review]

File: water.md [hunk 4]
```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
...
+
+Renewable water supplies are based on runoff estimates from Xanthos, a detailed global hydrology model ([Liu et al. 2017](water.html#liu2017); [Vernon et al. 2019](water.html#vernon2019)). Xanthos accounts for surface and subsurface processes to compute runoff at 0.5° grid resolution. Global climate datasets are utilized in conjunction with Xanthos to determine historical annual average runoff aggregated for each basin ([Turner et al. 2019b](water.html#turner2019b); [Liu et al. 2017](water.html#liu2017); [Vernon et al. 2019](water.html#vernon2019)). Of the total basin runoff, water available or accessible for human use takes into consideration requirements for ecosystem services, inaccessibility due to rapid flow and remote locations, and capacity of reservoir storage. Accessible fractions of total runoff vary across basins. Renewable water volumes up to the accessible fraction is available at nominal or low cost. Additional renewable water beyond the accessible fraction is available at a sharply higher cost to ensure availability of water for ecosystem services and to reflect capital investments necessary for reservoir expansion.
```
Comment (pralitp) [inline water.md]: Should this be the Liu 2019 reference?

**Review by pralitp — CHANGES_REQUESTED** [review]

File: water.md (hunk 3 above)
Comment (skim301) [reply water.md]: The purpose of the more detailed information serves to let the general audience know that we have thought about water supply issues in more detail and that we have the capability for addressing them. For the max resource, that info was important so let users know that we have a way to incorporate climate change impacts. I'd like to keep the more detailed documentation.

File: water.md (hunk 4 above)
Comment (skim301) [reply water.md]: Oops, it's the wrong Liu. It should have been the Yaling Liu paper from 2018. I will send you a revised water.md file with corrected reference. Thx.

**Review by pralitp — APPROVED** [review]

**Pull request merged**